packages/react/src/components/SuiteHeader/util/IdleTimer.js

```diff
@@ -77,6 +78,10 @@ class IdleTimer {
     )};expires=${expires};path=/;domain=${this.COOKIE_DOMAIN};`;
   }
 
+  deleteUserInactivityTimeout() {
```

Comment:
Perhaps call it `deleteUserInactivityTimeoutCookie`?

Reply:
Good idea, I'll change the method name. I didn't include `Cookie` in the name because we already have `setUserInactivityTimeout` and `updateUserInactivityTimeout`. So, perhaps I'll add the `Cookie` suffix to those as well.